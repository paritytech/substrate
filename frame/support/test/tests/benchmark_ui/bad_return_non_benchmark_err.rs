use frame_benchmarking::v2::*;
#[allow(unused_imports)]
use frame_support_test::Config;

#[benchmarks]
mod benches {
	use super::*;

	#[benchmark]
	fn bench() -> Result<(), BenchmarkException> {
		let a = 2 + 2;
		#[block]
		{}
		assert_eq!(a, 4);
		Ok(())
	}
}

fn main() {}
