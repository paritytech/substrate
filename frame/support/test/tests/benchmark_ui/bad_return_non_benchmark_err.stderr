error: expected `BenchmarkError`
  --> tests/benchmark_ui/bad_return_non_benchmark_err.rs:10:27
   |
10 |     fn bench() -> Result<(), BenchmarkException> {
   |                              ^^^^^^^^^^^^^^^^^^
